index.html

```diff
@@ -181,6 +191,48 @@ <h2 class="workheading">Multi-Post Stories</h2>
             <button type="button" class="projectButton">See Project</button>
         </div>
 
+        <footer class="footer-form">
+            <div class="footer-heading">
+                <h2 class="contact-heading">CONTACT US</h2>
```

Comment:
- [ OPTIONAL ] I recommend here adding some font weight to be exactly the same as the figma template so kindly update it 💯🚀🚀